views/about.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Since these tags are nested with the `html` tag, they should be indented one level